### PR TITLE
feat(Detail): add link to yarn.fyi in sidebar

### LINKED ISSUE #549: Add short permalink to package info page
https://yarnpkg.com/en/package/react should display `https://yarn.fyi/react` somewhere, so that people know that it works as a nice short permalink to the page. 

@Haroenv do you have any design ideas for this? Perhaps we could put it above the home page link in the right column.

### PR DESCRIPTION
**Pull request by Haroenv**
closes #549

screen shot 2017-06-27 at 10 19 23

http://deploy-preview-550--yarnpkg.netlify.com/en/package/yarn

### CODE REVIEW

**Comment by Haroenv** [comment]
Per @vvo's suggestion I added a copy button, but still a link inside that. This link stays on the same page, but links to yarn.fyi for consistency. 

screen shot 2017-06-27 at 15 41 50

**Comment by vvo** [comment]
Copy must include protocol (https://) no? right now it does not

File: _sass/_details.scss [hunk 1]
```diff
@@ -18,6 +18,25 @@
         width: 2em;
         opacity: .5;
       }
+
+      &__yarn {
+        display: flex;
+        align-items: center;
+
+        .copyable {
```
Comment (Daniel15) [inline _sass/_details.scss]: Is this styling actually specific to this use case, or could it be reused?

**Comment by Daniel15** [comment]
This looks pretty good to me! Could you make it show up as `yarn.fyi/{name}` (no `https://`), but copy `https://yarn.fyi/{name}` when clicked? Showing the full URL including `https://` looks slightly cluttered IMO, but we definitely need to copy it like @vvo mentioned.

**Comment by Daniel15** [comment]
Actually maybe `https://` is fine. I'll leave it up to you 😛

**Comment by Haroenv** [comment]
I don't have a general purpose way of copying something that's not visible, it might be a little confusing that you get copied something else than what is visible .

**Comment by Daniel15** [comment]
Sure. Feel free to merge this whenever you like. 😃

**Comment by Haroenv** [comment]
Weird things happen when the character count is a lot (e.g. [this-package-has-a-very-long-name-if-you-will-install-it-you-are-very-strong-lol](http://deploy-preview-550--yarnpkg.netlify.com/en/package/this-package-has-a-very-long-name-if-you-will-install-it-you-are-very-strong-lol))

For some reason the truncation happens too late and the icon is moved to the side before truncating.

screen shot 2017-06-27 at 19 16 05

**Comment by vvo** [comment]
https://clipboardjs.com/ has a way to copy something inside an attribute rather than the text.

**Comment by vvo** [comment]
> For some reason the truncation happens too late and the icon is moved to the side before truncating.

force the width of the text content to %?

**Comment by Daniel15** [comment]
> has a way to copy something inside an attribute rather than the text.

clipboardjs.com likely takes the same approach as we take at Facebook - Create a hidden element and copy text from it:
```js
let success;
const node = document.createElement('textarea');
document.body.appendChild(node);
node.value = text;
node.select();
try {
  success = document.execCommand('copy');
} catch (e) {
  success = false;
}
root.removeChild(node);
```

**Comment by Haroenv** [comment]
I didn't know node.select() was now supported, will remember that in the future 😄

**Pull request merged**